src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_PER_DAY = 40;
+  const TOTAL = days * RENT_PER_DAY;
+  const THREE_DAYS_DISCOUNT = 20;
+  const SEVEN_DAYS_DISCOUNT = 50;
+  const SEVEN_DAYS = 7;
+  const THREE_DAYS = 3;
+
+  if (days < THREE_DAYS) {
+    return days * RENT_PER_DAY;
```

Comment:
There's no need to recalculate the total cost here since you have already calculated it in line 8. You can return 'TOTAL' directly.